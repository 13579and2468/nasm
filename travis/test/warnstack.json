[
	{
		"description": "Test warning stack",
		"id": "warnstack",
		"source": "warnstack.asm",
		"option": "-E",
		"target": [
			{ "stdout": "warnstack.stdout" },
			{ "stderr": "warnstack.stderr" }
		]
	}
]
