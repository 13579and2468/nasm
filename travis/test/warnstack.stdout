%line 2+1 ./travis/test/warnstack.asm
 [warning push]
 [warning -user]

 [warning pop]

 [warning -user]

 [warning pop]